hack/build-manifests.sh

```diff
@@ -421,6 +421,10 @@ rm -f ${CRD_DIR}/*
 cp -f ${TEMPDIR}/*.${CRD_EXT} ${CRD_DIR}
 cp -f ${TEMPDIR}/*.${CRD_EXT} ${CSV_DIR}
 
+# Validate the yaml files
+(cd ${CRD_DIR} && docker run -it --rm -v "$(pwd)":/yaml sdesbure/yamllint yamllint -d "{extends: relaxed, rules: {line-length: disable}}" *.yaml)
+(cd ${CSV_DIR} && docker run -it --rm -v "$(pwd)":/yaml sdesbure/yamllint yamllint -d "{extends: relaxed, rules: {line-length: disable}}" *.yaml)
```

Comment:
why not using the default (more strict) lint configuration and just relax the indentation strict spacing rule?
i.e.
```
sdesbure/yamllint yamllint -d "{extends: default, rules: {line-length: disable, indentation: {indent-sequences: consistent}}}" *.yaml
```

That way we can catch more errors on yamls